README.md

```diff
@@ -242,7 +242,16 @@ Please remember to use your regional currency when interpolating strings.
 const const name = "world"!
 print("Hello ${name}!")!
 print("Hello £{name}!")!
-print("Hello €{name}!")!
+
+// Follow the local typographical norms:
+print("Hello {name} €!")!
```

Comment:
```suggestion
\```

Make sure you follow your local typographical norms.

```java
print("Hello {name}€!")!
```

^ I think this is pretty fun, so we could give it some more breathing room. (Note-to-self: Might need to remove my backslash)

Reply:
Sounds good!

Backslash seems to mangle it, yeah – if you get rid of it I can commit this (or I can just copy it over myself)

Reply:
I wonder what happens if you click 'Commit suggestion' now. I don't think github PRs were designed for this :)

Reply:
pow!